README.md

```diff
@@ -1,110 +1,148 @@
 
-Say you want to create a timestamp of the current time in the UTC+0 timezone.
+## Quick Start
 
 ```python
-from heliclockter import datetime_utc
+from heliclockter import datetime_utc, datetime_local, datetime_tz
 
-now = datetime_utc.now()
+# UTC datetime
+utc_now = datetime_utc.now()
 # datetime_utc(2022, 11, 4, 15, 28, 10, 478176, tzinfo=zoneinfo.ZoneInfo(key='UTC'))
-```
 
-Or imagine you want to create a timestamp 2 hours in the future from now:
+# Local timezone datetime  
+local_now = datetime_local.now()
 
-```python
-from heliclockter import datetime_utc
+# Any timezone datetime
+paris_tz = datetime_tz.now("Europe/Paris")
+
+# Create a timestamp 2 hours in the future
+future = datetime_utc.future(hours=2)
 
-two_hours_from_now = datetime_utc.future(hours=2)
-# datetime_utc(2022, 11, 4, 17, 28, 52, 478176, tzinfo=zoneinfo.ZoneInfo(key='UTC'))
+# Parse strings (naive timestamps assumed UTC)
+parsed = datetime_utc.strptime('2022-11-04T15:49:29', '%Y-%m-%dT%H:%M:%S')
 ```
 
-Features
---------
+## Why heliclockter?
 
-* Runtime enforcable timezone-aware datetimes
-* Utilities for instantiating, mutating and serializing timezone-aware datetimes
-* Statically type check-ble
-* Pydantic integration
-* Extensive test suite
-* No third party dependencies
+Python's standard `datetime` allows "naive" datetimes without timezone info, leading to bugs when:
+- Mixing naive and aware datetimes (causes runtime TypeErrors)
+- Deploying across different timezones
+- Forgetting to add `tzinfo` when creating datetimes
 
-Installation
-------------
+`heliclockter` enforces timezone-aware datetimes at the type level, catching these issues before production.
 
-To install `heliclockter`, simply: 
+## Key Features
 
-    $ pip install heliclockter
+- **Always timezone-aware** - No more naive datetime accidents
+- **Type safe** - Full typing support for better IDE experience  
+- **Zero dependencies** - Lightweight, uses only standard library
+- **Pydantic support** - Automatic integration when Pydantic is installed
+- **Python 3.9+** - Modern Python for modern applications
 
-More examples
--------------
+## Examples
 
-Imagine you want to parse a JSON response from a third party API which includes a timestamp, and you
-want to handle the timestamp in the UTC+0 timezone regardless of how the 3rd party relays it. This 
-can easily be done with `pydantic` and `heliclockter`:
+### Timezone conversions
 
 ```python
-import requests
-from pydantic import BaseModel
 from heliclockter import datetime_utc
+from zoneinfo import ZoneInfo
 
+# Start with UTC
+utc_time = datetime_utc.now()
 
-class ApiResponse(BaseModel):
-    current_time: datetime_utc
-
+# Convert to Tokyo time
+tokyo_time = utc_time.astimezone(ZoneInfo('Asia/Tokyo'))
 
-def get_response() -> ApiResponse:
-    response = requests.get('https://some-api.com/time')
-    return ApiResponse.parse_obj(response.json())
+# Convert to New York time
+ny_time = utc_time.astimezone(ZoneInfo('America/New_York'))
```

Comment:
These conversions don't work as the class is still a `datetime_utc`, which only accepts the utc timezone. One would need to implement `datetime_tokyo` and `datetime_new_york` classes and call e.g. `datetime_tokyo.from_datetime(utc_time)`. I would then also move this example down below where you give the example of the custom class.

This might not be intuitive, thinking about it, but that is how it works at the moment.